doc/chapter-firststeps.asciidoc

```diff
@@ -52,10 +52,6 @@ provide username/password:
 
 	https://username:password@hostname.domain.tld/feed.rss
 
-In case there is a _@_ in the username, you need to write it as _%40_.
```

Comment:
Please restore the first sentence about the percent encoding, that's vital. Only the detail explanation below can be dropped.

Reply:
Oops, that makes sense! Added it back